examples/README.md

```diff
@@ -0,0 +1,233 @@
+# oneAPI Math Kernel Library (oneMKL) Interfaces Examples 
+oneAPI Math Kernel Library (oneMKL) Interfaces offers examples with the following routines: 
+- blas: level3/gemm_usm  
+- rng: uniform_usm  
+- lapack: getrs_usm
+
+Each routine has one run-time dispatching example and one compile-time dispatching example (which uses both mklcpu and cuda backends), located in `example/<$domain>/run_time_dispatching` and `example/<$domain>/compile_time_dispatching` subfolders, respectively.
+
+To build examples, use cmake build option `-DBUILD_EXAMPLES=true`.  
+Compile_time_dispatching will always be built if `-DBUILD_EXAMPLES=true`.   
+Run_time_dispatching will be build if `-DBUILD_EXAMPLES=true` and `-DBUILD_SHARED_LIBS=true`
+
+The example executable naming convention follows `example_<$domain>_<$routine>_<$backend>` for compile-time dispatching examples 
+  or `example_<$domain>_<$routine>` for run-time dispatching examples. 
+  E.g. `example_blas_gemm_usm_mklcpu_cublas `  `example_blas_gemm_usm`
+
+## Test outputs (blas, rng, lapack)
+  
+## blas
+
+Run-time dispatching examples with mklcpu backend
+```
+$ export SYCL_DEVICE_FILTER=cpu
```

Comment:
can we add SYCL_DEVICE_FILTER=gpu as a comment to show how to use mklgpu backend as well?

Reply:
added mklgpu output